js/directives/lucca/user-picker.js

```diff
@@ -29,7 +29,7 @@
 	}]; // MAGIC LIST OF PROPERTIES
 
 	var uiSelectChoicesTemplate = "<ui-select-choices position=\"down\" repeat=\"user in users\" refresh=\"find($select.search)\" refresh-delay=\"0\" ui-disable-choice=\"!!user.overflow\">" +
-	"<div ng-bind-html=\"user.firstName + ' ' + user.lastName | highlight: $select.search\" ng-if=\"!user.overflow\"></div>" +
+	"<div>{{user.firstName}} {{user.lastName}} <span class=\"lui label secondary\" ng-if=\"user.info\">{{user.info}}</span></div>" +
```

Comment:
you should create your own filter that creates some html and uses highlight, something like
```js
.filter('luifHighlight', ['$filter', function($filter) {
return function(_input, _clue, _info) {
  return $filter('highlight')(_input, _clue) + (!!_info ? "<span class='lucca label'>" + _info + "</span>");
}
}]);
```
if you do that you can then use
```html
<div ng-bind-html="user.firstName + ' ' + user.lastName | luifHighlight : $select.search : user.info"></div>
```